engine/swarm/services.md

```diff
@@ -394,40 +394,28 @@ option to the `--publish` flag.
 > `--mode=global` flag on `docker service  create`, it will be difficult to know
 > which nodes are running the service in order to route work to them.
 
-##### Example: Run a `cadvisor` monitoring service on every swarm node
-
-[Google cAdvisor](https://hub.docker.com/r/google/cadvisor/) is a tool for
-monitoring Linux hosts which run containers. Typically, cAdvisor is run as a
-stand-alone container, because it is designed to monitor a given Docker Engine
-instance. If you run cAdvisor as a service using the routing mesh, connecting
-to the cAdvisor port on any swarm node will show you the statistics for
-(effectively) **a random swarm node** running the service. This is probably not
-what you want.
-
-The following example runs cAdvisor as a service on each node in your swarm and
-exposes cAdvisor port locally on each swarm node. Connecting to the cAdvisor
-port on a given node will show you **that node's** statistics. In practice, this
-is similar to running a single stand-alone cAdvisor container on each node, but
-without the need to manually administer those containers.
+##### Example: Run a `nginx` web server service on every swarm node
+
+[nginx](https://hub.docker.com/_/nginx/) is an open source reverse proxy, load balancer, HTTP cache, and a web server. If you run nginx as a service using the routing mesh, connecting
+to the nginx port on any swarm node will show you the web page for
+(effectively) **a random swarm node** running the service.
+
+The following example runs nginx as a service on each node in your swarm and
```

Comment:
Is it really necessary for this example? I can't find when having a nginx listening locally would be a good idea.